trtllm-c/logits.cpp

```diff
@@ -201,8 +207,8 @@ static void logitsPostProcessorFn(std::vector<tle::IdType> const& reqIds, std::v
     if (dp > 0)
     {
         cudaMemcpyAsync(cudaMasksData, masksData, max_offset + mask_stride, cudaMemcpyHostToDevice, stream);
-        mask_logits_ext(cudaLogitPtrs, cudaMasksOffsets, mask_fractions_offset, temperatures_offset, dp, nVocab,
-            mask_stride / 4, tp, stream);
+        mask_logits_ext(cudaLogitPtrs, cudaMasksOffsets, mask_fractions_offset, temperatures_offset, ln_min_p_offset,
```

Comment:
we should probably pass an array of structs with mask offset, temp, and min_p - it's getting too many for comfort ; but you can keep it as is for now